lib/screens/onboarding/identity_verification/onboarding_reference_account_iban.dart

```diff
@@ -0,0 +1,307 @@
+                      IvoryTextField(
+                        controller: _accountIbanController,
+                        label: 'Reference account IBAN',
+                        placeholder: 'E.g. DE11110101010100000020',
+                        onChanged: (value) {
+                          if (value != 'DE11110101010100000020') {
+                            _accountIbanController
+                                .setErrorText('IBAN not found. Please try again with a different one.');
+                          } else {
+                            _accountIbanController.setErrorText(null);
+                          }
+                        },
+                      ),
+                      const SizedBox(height: 24),
+                      const Spacer(),
+                      ApprovePolicy(
+                        isChecked: _isChecked,
+                        isDisabled: _continueButtonController.isLoading,
+                        onChanged: (checked) {
+                          setState(() {
+                            _isChecked = checked;
+                            _isValidated();
+                          });
+                        },
+                        message: Text.rich(
+                          style: ClientConfig.getTextStyleScheme().bodyLargeRegular,
+                          TextSpan(
+                            children: [
+                              const TextSpan(text: 'I agree to the '),
+                              TextSpan(
+                                  text: 'Swisscom\'s Terms & Conditions',
+                                  style: ClientConfig.getTextStyleScheme()
+                                      .bodyLargeRegularBold
+                                      .copyWith(color: ClientConfig.getColorScheme().secondary)),
+                              const TextSpan(text: '.'),
+                            ],
+                          ),
+                        ),
+                      ),
+                      const SizedBox(height: 16),
+                      ListenableBuilder(
+                        listenable: _continueButtonController,
+                        builder: (context, child) => PrimaryButton(
+                            text: "Continue to verification",
+                            isLoading: _continueButtonController.isLoading,
+                            onPressed: _continueButtonController.isEnabled
+                                ? () {
+                                    StoreProvider.of<AppState>(context).dispatch(
+                                      CreateUrlForIntegrationCommandAction(
+                                        accountName: _accountNameController.text,
+                                        iban: _accountIbanController.text,
+                                      ),
+                                    );
+
+                                    // Navigator.of(context).pushNamed(NextScreen.routeName);
```

Comment:
This comment is not needed, as the navigation to the next step will be done in the `StoreConnector.onWillChange(...)`